lib/notifier.js

```diff
@@ -295,6 +296,40 @@ function buildItemData(item, callback) {
 }
 
 
+// Is the error level of a given item greater than or equal to the configured
+// minimum level?
+function levelGteMinimum(item) {
+  var levels, messageLevel, i, length;
+
+  levels = [
+    'critical',
+    'error',
+    'warning',
+    'info',
+    'debug'
+  ];
+  messageLevel = item.payload.level === undefined ? 'error' : item.payload.level;
+
+  for (i = 0, length = levels.length; i < length; i++) {
+    if (levels[i] === messageLevel) {
+      return true;
+    }
+    if (levels[i] === SETTINGS.minimumLevel) {
+      return false;
+    }
+  }
+
+  // At this point the minimum level was never reached and the message level
+  // wasn't a known one; something is wrong
+  logger.error('[Rollbar] minimumLevel of "%s" is unknown; '
```

Comment:
I knew I was forgetting something and the rebase had more to it last time. Thanks.